python_modules/dagster/dagster/core/materializable.py

```diff
@@ -0,0 +1,55 @@
+import json
+
+from dagster import check
+
+from .configurable import (
+    Field,
+    ConfigurableSelectorFromDict,
+)
+
+
+def define_path_dict_field():
+    from .types import (
+        Dict,
+        Path,
+    )
+    return Field(Dict({'path': Field(Path)}))
+
+
+class Materializeable(object):
+    def define_materialization_config_schema(self):
+        check.failed('must implement')
+
+    def materialize_runtime_value(self, _config_spec, _runtime_value):
+        check.failed('must implement')
+
+
+class MaterializeableValueConfigSchema(ConfigurableSelectorFromDict):
+    def __init__(self):
+        # TODO: add pickle
+        super(
+            MaterializeableValueConfigSchema,
+            self,
+        ).__init__(fields={'json': define_path_dict_field()})
+
+    def iterate_types(self):
+        return []
+
+
+class MaterializeableValue(Materializeable):
```

Comment:
something about this makes me a little uncomfortable, I think if we called it DagsterBuiltinMaterializable or something I would feel better -- as this isn't really intended to be inherited by user classes, we want them to write their own implementations of Materializable